src/measure_theory/measure/finite_measure_weak_convergence.lean

```diff
@@ -154,6 +158,246 @@ def coe_add_monoid_hom : finite_measure α →+ measure α :=
 instance {α : Type*} [measurable_space α] : module ℝ≥0 (finite_measure α) :=
 function.injective.module _ coe_add_monoid_hom finite_measure.coe_injective coe_smul
 
+variables [topological_space α]
+
+/-- The pairing of a finite (Borel) measure `μ` with a nonnegative bounded continuous
+function is obtained by (Lebesgue) integrating the (test) function against the measure. This
+is `finite_measure.test_against'`. -/
+abbreviation test_against_nn (μ : finite_measure α) (f : α →ᵇ nnreal) : ℝ≥0 :=
+(lintegral (μ : measure α) ((coe : ℝ≥0 → ℝ≥0∞) ∘ f)).to_nnreal
```

Comment:
```suggestion
(∫⁻ x, f x ∂(μ : measure α)).to_nnreal
```
I don't think you need to be explicit about the coercion here

Reply:
I also think we should relax the condition on `f` to be only a function and only restrict them to be bounded and continuous whenever necessary.

Reply:
I understand mathlib aims for great generality, and I have even been given convincing practical reasons why. This `test_against_nn`, however, is only a shorthand for the integral (which is general), to be used for the very specific purpose of defining the topology, so I felt it is not one of the cases where striving for generality is meaningful (the `lintegral` is already there for the general use). One of the arguments in favor of generality, namely avoiding unnecessary checks by the automation, is still valid, though. So although it feels wrong mathematically and the gain in generality is not useful in view of the already general `lintegral`, even that one argument may be enough, and I should remove the continuity and boundedness assumptions --- at least if the coercion of a `bounded_continuous_function` to just a function is sufficiently automatic.